server/.env.example

```diff
@@ -67,31 +67,33 @@ APPLICATIONINSIGHTS_CONNECTION_STRING=
 ###############################################################
 ##      Testing (Comment out ALL the above variables)        ##
 ###############################################################
-# TEST_ENV=
+# TEST_ENV=true
 
-# PORT=
-# NODE_OPTIONS=
+# PORT=5002
+# NODE_OPTIONS=--trace-deprecation
 
-# JWT_SECRET_KEY=
+# JWT_SECRET_KEY=testingSecretKey
 
-# CLIENT_URL=
-# SERVER_URL=
+# CLIENT_URL=http://localhost:3001
+# SERVER_URL=http://localhost:5002
 
-# SENDGRID_API_KEY=
-# EMAIL_SENDER=
-# EMAIL_PUBLIC_COMMENT_LA_CITY=
-# EMAIL_PUBLIC_COMMENT_WEB_TEAM=
+# SENDGRID_API_KEY=SG.testAPIkey
```

Comment:
Does this mean tdm-server-tests can run with .env.example?

Reply:
Nope! I just changed it because VSCode has better highlighting with the new name. Also included the variables so new members could run the tests easily without the formal variables. I tried to get this to work unsuccessfully with `./push.sh` 
An alternate strategy I thought of this morning is a function in the push script that can uncomment the test variables for the server tests and then re-comment upon completion. I'll have to check if this is viable, but editing the .env file with code feels unkosher. Thoughts?

Reply:
I was looking into that but had some concerns.

Since the container has a single database that is [restored before every test suite](https://github.com/hackforla/tdm-calculator/blob/develop/server/_jest-setup_/local-setup/setup-after-env.js) (i.e. before every test file), the logic that handles this will call `tedious-pool.js` the [connection logic](https://github.com/hackforla/tdm-calculator/blob/develop/server/app/services/tedious-pool.js) from the server which will consume the original .env file. this means that this file would require a conditional to see which `.env` to consume right? Is this crufty?

Reply:
It means everywhere we use dotenv should probably go through a tdmenv wrapper.

Reply:
so new issue for refactoring codebase with this ^